File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/entities/adventurers/units/EternalFortress.java
package it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.units;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.Skills;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.PotionDrinkerType;

/* JADX INFO: loaded from: classes3.dex */
public class EternalFortress extends Adventurer {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer
    protected void configureStatistics() {
        this.maxLevel = 45;
        this.baseMaxHp = 380;
        this.baseConstitution = 40;
        this.baseIntelligence = 20;
        this.baseDexterity = 12;
        this.baseDefense = 30;
        this.baseMagicDefense = 30;
        this.threat = 5;
        this.imageId = R.drawable.unit_eternal_fortress;
        this.idName = R.string.adventurer_eternal_fortress_name;
        this.idDescription = R.string.adventurer_eternal_fortress_description;
        this.passiveSkill = Skills.PASSIVE_THREATENING_IV;
        this.activeSkill = Skills.ACTIVE_TAUNT_IV;
        this.weaponType = R.string.type_sword;
        this.armorType = R.string.type_armor_heavy;
        this.potionDrinkerType = PotionDrinkerType.WARRIOR;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/entities/adventurers/units/EternalFortress.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.units

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.Skills
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.PotionDrinkerType

class EternalFortress : Adventurer() {
    override fun configureStatistics() {
        maxLevel = 45
        baseMaxHp = 380
        baseConstitution = 40
        baseIntelligence = 20
        baseDexterity = 12
        baseDefense = 30
        baseMagicDefense = 30
        threat = 5
        imageId = R.drawable.unit_eternal_fortress
        idName = R.string.adventurer_eternal_fortress_name
        idDescription = R.string.adventurer_eternal_fortress_description
        passiveSkill = Skills.PASSIVE_THREATENING_IV
        activeSkill = Skills.ACTIVE_TAUNT_IV
        weaponType = R.string.type_sword
        armorType = R.string.type_armor_heavy
        potionDrinkerType = PotionDrinkerType.WARRIOR
    }
}
